test_databaseconnect.py

```diff
@@ -0,0 +1,151 @@
+import databaseconnect
+import config
+import pytest
+import os
+import dotenv
+import mysql.connector  as mysql
+
+dotenv.load_dotenv()
+
+class TestClass:
+    
+    @pytest.fixture(scope="session", autouse=True)
+    def setup_init(self):
+        # Will be executed before the first test
+        config.database = "test"
+        try:
+            test_db = mysql.connect(
+                host=config.host,
+                user=config.user,
+                passwd=config.password
+            )
+
+            cursor = test_db.cursor()
+            cursor.execute("CREATE DATABASE {}".format(config.database))
+            print("test database created")
+
+        except Exception:
+            print ("Failed to create test database")
+            # rolling back to orignal db
+            config.database = os.getenv("mapbotdatabase")
```

Comment:
Would like to point out ([again](https://github.com/vishakha-lall/MapBot/pull/103/files#r398357305)), this will not work
```python
config.database = os.getenv("mapbotdatabase")
```
Since #104 is merged, environment variables have different names.
> Changing config.database outside of config.py for tests may cause inconsistencies.

@janakrajchadha @msk4862 @vishakha-lall

Reply:
okay